scripts/runner.py

```diff
@@ -49,37 +49,48 @@ def log_data(refresh_rate, logger_list, client, farm_name):
     )
 
     logging.info("DB Client Configured")
-
     farm_name = farm.get_name()
 
-    for sensor in sensors:
-        refresh_rate = sensor.get_refresh()
+    for sensor in sensors:  # NetworkSensors will have a refresh rate of None
         logger = InfluxDBLogger(sensor)
-        sensor.set_logger(logger)
         logger.set_location(sensor.get_location())
-        logger.set_refresh_rate(refresh_rate)
-        if refresh_rate not in loggers:
-            loggers[refresh_rate] = [logger]
-        else:
-            loggers.get(refresh_rate).append(logger)
+        sensor.set_logger(logger)
+        if isinstance(sensor, NetworkSensor):
+            sensor.set_influxconnection(db_client)
+        elif not isinstance(sensor, NetworkSensor):
+            refresh_rate = sensor.get_refresh()
+            logger.set_refresh_rate(refresh_rate)
+
+        if logger.get_refresh_rate() is not None:
+            if logger.get_refresh_rate() not in loggers:
+                loggers[refresh_rate] = [logger]
+            else:
+                loggers.get(refresh_rate).append(logger)
 
     logging.info("Loggers created for sensors")
     threads = []
 
     for refresh_rate, logger_list in loggers.items():
         thread = threading.Thread(
-            target=log_data, args=(refresh_rate, logger_list, db_client, farm_name)
+            target=log_data,
+            args=(refresh_rate, logger_list, db_client),
+            daemon=True,
         )
         threads.append(thread)
-
+    print("Threads: ", len(threads))

```

Comment:
Can you change this print statement to a logging statement? Perhaps merge with the statement below